src/sensor/ping360.cpp

```diff
@@ -498,15 +497,17 @@ void Ping360::resetSettings()
 {
     qCDebug(PING_PROTOCOL_PING360) << "Settings will be reseted.";
 
-    _gain_setting = _firmwareDefaultGainSetting;
+    // There is no angle control
     _angle = _firmwareDefaultAngle;
-    _transmit_duration = _firmwareDefaultTransmitDuration;
-    _sample_period = _firmwareDefaultSamplePeriod;
-    _transmit_frequency = _firmwareDefaultTransmitFrequency;
-    _num_points = _firmwareDefaultNumberOfSamples;
-    deltaStep(_angle_offset, false);
+    // This variables should not be set directly without equality checks,
+    // such thing will result in a miss signal emition
+    set_gain_setting(_firmwareDefaultGainSetting);
+    set_transmit_duration(_firmwareDefaultTransmitDuration);
```

Comment:
This does not fully address #781 
There are two things remaining:
1. set the sonar to 2 meter range, not 1.5m (we want to use the _application_ default settings. not the _firmware_ default settings.)
2. delete line 501, do not update the angle